test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -39,6 +39,8 @@ export default class PerfCompare extends Component {
             testID: "",
         },
         selectedRuns: {
+        	baselineServerURL: "",
```

Comment:
Can you please correct the indentation for these 2 lines?

Reply:
Fixed it, thank you.